extensions/binding/org.eclipse.smarthome.binding.astro/ESH-INF/config/config.xml

```diff
@@ -6,12 +6,9 @@
 
 	<config-description uri="thing-type:astro:config">
 		<parameter name="geolocation" type="text" required="true">
+			<context>location</context>
 			<label>GeoLocation</label>
```

Comment:
Could we fix the label as well? "GeoLocation" is not a word... Probably best to change it to "Location".